ingestion/src/metadata/ingestion/sink/metadata_rest.py

```diff
@@ -387,6 +389,22 @@ def _bootstrap_entities(self):
         team_response = self.api_client.get("/teams")
         for team in team_response["data"]:
             self.team_entities[team["name"]] = team["id"]
+        role_response = self.api_client.get("/roles")
+        for role in role_response["data"]:
+            self.role_entities[role["name"]] = role["id"]
+
+    def _create_role(self, role: EntityReference) -> None:
+        metadata_role = CreateRoleEntityRequest(
+            name=role.name, displayName=role.name, description=role.description
+        )
+        try:
+            r = self.metadata.create_or_update(metadata_role)
+            instance_id = str(r.id.__root__)
+            self.role_entities[role.name] = instance_id
+        except Exception as err:
+            logger.error(traceback.format_exc())
```

Comment:
can we change this to debug and one below

Reply:
Done